README.md

```diff
@@ -1,2 +1,18 @@
 # open-iot-stem
 One Planet Education Network IoT STEM Education Kit
+
+
+Arduino work through for Rak 4630/4631
+For all this to work in Arduino you need to hav:
+
+Adafruit nrf utility installed through PIP (Python3).
+https://github.com/adafruit/Adafruit_nRF52_nrfutil
+
+All of the Adafruit board definitions in the sketch through the library manager.
+
+The bsec library needs to be at (1.4.703).
+
+As well the Adafruit Bluefruit nrf51 library through the library manager.
+
+Specifically the Rak nrf (0.21.20) board version.
+#https://forum.rakwireless.com/t/just-updated-nrf-boards-for-rak-4631-broke-my-code/5189/2
```

Comment:
Why does this URL line should not start with `#`?

Reply:
Makes it one size bigger so it stands out more

Reply:
dose not half to

Reply:
OK, I see you changed it. let's go with it now and we can re-style later.